modules/rostests/winetests/user32/msg.c

```diff
@@ -15182,6 +15182,25 @@ static const struct message wm_popup_menu_3[] =
     { 0 }
 };
 
+static const struct message wm_popup_menu_4[] =
+{
+    { HCBT_CREATEWND, hook },
+    { WM_ENTERMENULOOP, sent|wparam|lparam, 0, 0 },
+    { WM_INITMENU, sent|lparam, 0, 0 },
+    { WM_INITMENUPOPUP, sent|lparam, 0, 0x10000 },
+    { HCBT_KEYSKIPPED, hook|wparam|lparam|optional, VK_DOWN, 0x10000001 },
+    { WM_MENUSELECT, sent, MAKEWPARAM(0,0xffff), 0 },
+    { HCBT_KEYSKIPPED, hook|wparam|lparam|optional, VK_DOWN, 0xd0000001 },
+    { HCBT_KEYSKIPPED, hook|wparam|lparam|optional, VK_ESCAPE, 0x10000001 },
+    { HCBT_DESTROYWND, hook },
+    { WM_UNINITMENUPOPUP, sent|lparam, 0, 0x20000000 },
+    { WM_MENUSELECT, sent, MAKEWPARAM(0,0xffff), 0 },
+    { WM_EXITMENULOOP, sent|wparam|lparam, 0, 0 },
+    { HCBT_KEYSKIPPED, hook|wparam|lparam|optional, VK_ESCAPE, 0xc0000001 },
+    { WM_KEYUP, sent|wparam|lparam, VK_ESCAPE, 0xc0000001 },
+    { 0 }
+};
```

Comment:
This is 3rd party Wine code, changes should be sent upstream. Unless you move the changes into `modules/rostests/apitests/user32/` that belongs to us.